debian/matrix-synapse.init

```diff
@@ -50,6 +51,9 @@ get_config_key()
 #
 do_start()
 {
+	# Fail silently if CONFIGFILE_SERVERNAME doesn't exist
+	[ -f $CONFIGFILE_SERVERNAME ] || return 0
```

Comment:
Why fail silently? Don't we want to yell loudly?

Reply:
There are many situations when the package is left unconfigured for some reason, and failing here may result in some inconveniences. Here’s one I can think of right now: the user cancels the configuration e.g. by dismissing the debconf dialogue, the file doesn’t get created, dpkg attempts to start the daemon and fails, apt fails too, the user is left with an error message.

At best, I think, we should display a warning here (something like: `Missing configuration bits, not starting`).